tests/unit_tests/base.py

```diff
@@ -95,3 +97,15 @@ def setUpClass(cls):
     @classmethod
     def tearDownClass(cls):
         cls.table.delete()
+
+    def assertCors(self, response, expected_allowed_methods, check_origin=True):  # pylint: disable=invalid-name
+        if check_origin:
+            self.assertIn('Access-Control-Allow-Origin', response.headers)
+            self.assertTrue(
+                re.match(ALLOWED_DOMAINS_PATTERN, response.headers['Access-Control-Allow-Origin'])
+            )
+        self.assertIn('Access-Control-Allow-Methods', response.headers)
+        self.assertListEqual(
+            sorted(expected_allowed_methods),
+            sorted(response.headers['Access-Control-Allow-Methods'].split(','))
```

Comment:
In other services, `sorted(
                map(
                    lambda m: m.strip(),
                    response.headers['Access-Control-Allow-Methods'].split(',')
                )
            )` is used here instead. Don't know the exact reasoning behind this, but probably this needs to be changed here, too?

Reply:
This is because in other services the header is set with spaces after the comma `GET, HEAD, OPTIONS` so for the check we need to strip the spaces. Here in the method that construct the header value, there is no space after the comma. That's the reason. From the standard both are correct.